spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -24,8 +24,8 @@
   let(:callback_url)        { "http://#{request_host}/v1/sessions/callback" }
   let(:logout_redirect_url) { 'http://127.0.0.1:3001/logout/' }
 
-  let(:settings_no_context) { build(:settings_no_context, assertion_consumer_service_url: callback_url) }
-  let(:rubysaml_settings)   { build(:rubysaml_settings, assertion_consumer_service_url: callback_url) }
+  let(:settings_no_context) { build(:settings_no_context_v1, assertion_consumer_service_url: callback_url) }
```

Comment:
I just realized that `:settings_no_context` isn't used anywhere in `sessions_controller_spec.rb` 😬